src/ray/util/env.cc

```diff
@@ -39,4 +42,13 @@ void UnsetEnv(const std::string &name) {
   RAY_CHECK_EQ(ret, 0) << "Failed to unset env var " << name;
 }
 
+bool IsEnvSet(const std::string &name) {
```

Comment:
the naming is confusing, I'd expect it only checks whether the env is set or not, instead of checking the actual value.